src/tmx/TmxApi/CMakeFiles/tmxapi.dir/C.includecache

```diff
@@ -0,0 +1,5234 @@
+#IncludeRegexLine: ^[ 	]*[#%][ 	]*(include|import)[ 	]*[<"]([^">]+)([">])
+
+#IncludeRegexScan: ^.*$
+
+#IncludeRegexComplain: ^$
+
+#IncludeRegexTransform: 
+
+/home/anjanrayamajhi/Projects/V2X-CARMACloud/V2X-Hub/src/tmx/TmxApi/tmx/IvpMessage.c
```

Comment:
This file contains specific username home path. Can we remove this cache file? or git ignore it?